### PR TITLE
Config reader

### PR DESCRIPTION
**Pull request by grawix**

### CODE REVIEW

File: Framework/BrowserSettings/ConfigReader.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Framework.BrowserSettings
+{
+    class ConfigReader
+    {
+        protected static IConfiguration config;
```
Comment (VolodymyrRomanyshyn) [inline Framework/BrowserSettings/ConfigReader.cs]: config is never used and always will be null

File: Framework/BrowserSettings/ConfigReader.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Framework.BrowserSettings
+{
+    class ConfigReader
```
Comment (VolodymyrRomanyshyn) [inline Framework/BrowserSettings/ConfigReader.cs]: Can we use this class on model settings?

Comment (grawix) [reply Framework/BrowserSettings/ConfigReader.cs]: Yes, class is public and method is static. Class could be static also, but we need to discuss

File: Framework/BrowserSettings/ConfigReader.cs (hunk 1 above)
Comment (grawix) [reply Framework/BrowserSettings/ConfigReader.cs]: Fixed. Value will be assigned when `Config ` is invoked first time.

**Pull request merged**